src/components/input/_macro.njk

```diff
@@ -70,7 +71,7 @@
 
         {% if params.prefix is defined or params.suffix is defined %}
             {% if params.prefix is defined %}
-                {% set prefixClass = " input-type--prefix" %}
+                {% set prefixClass = " input-type--prefix u-mb-s" %}
```

Comment:
Could probably use `@extend .u-mb-s;` in the sass rather than setting and adding it to the template?